gradle/dependencies.gradle

```
@@ -41,6 +42,20 @@ if (hasProperty('scalaVersion')) {
   versions["scala"] = defaultScala212Version
 }
 
+if (hasProperty('streamsScalaVersion')) {
+  if (streamsScalaVersion == '2.12') {
+    versions["streamsScala"] = defaultScala212Version
+  } else if (streamsScalaVersion == '2.13') {
+    versions["streamsScala"] = defaultScala213Version
+  } else if (streamsScalaVersion == '3') {
+    versions["streamsScala"] = defaultScala3Version
+  } else {
+    versions["streamsScala"] = streamsScalaVersion
+  }
+} else {
+  versions["streamsScala"] = defaultScala212Version
```

This should be something like 
```suggestion
  versions["streamsScala"] = versions["scala"]
```
If you fail to specify `streamsScalaVersion` you should have the same version as `scalaVersion`.